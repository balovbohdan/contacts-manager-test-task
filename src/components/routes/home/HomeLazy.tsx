import * as React from 'react';

const {lazy, Suspense} = React;

export default () => {
    const C = lazy(() => import('./Home'));

    return (
        <Suspense fallback='wait please'>
            <C/>
        </Suspense>
    );
};