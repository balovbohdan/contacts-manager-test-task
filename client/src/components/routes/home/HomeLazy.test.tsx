import * as React from 'react';
import {shallow} from 'enzyme';

import HomeLazy from './HomeLazy';

describe('Home route, lazy version', () => {
    it('Should match snapshot', () => {
        const home = shallow(<HomeLazy/>);

        expect(home).toMatchSnapshot();
    });
});