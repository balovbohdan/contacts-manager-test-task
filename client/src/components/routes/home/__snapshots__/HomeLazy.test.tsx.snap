// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home route, lazy version Should match snapshot 1`] = `
<Suspense
  fallback={<Unknown />}
>
  <lazy />
</Suspense>
`;
